build-tools/biome.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Biome only supports json configs, I take it? Are they okay with comments in the json?

Reply:
They don't support jsonc yet, but there's an issue: https://github.com/biomejs/biome/issues/1259